projects/core/src/lib/layer-collections/layer-collection-layer/layer-collection-layer.component.scss

```diff
@@ -4,17 +4,18 @@
 }
 
```

Comment:
The details will get a scroll bar if the sidenav gets to small. I think this is fine. However there is a bit missing on the right...

![image](https://user-images.githubusercontent.com/1027102/221585621-2e4a3c00-6f1e-4287-967d-9cd4c3882112.png)

maybe you could look into that?

Reply:
[be126a1](https://github.com/geo-engine/geoengine-ui/pull/599/commits/be126a16043a28e1a8b742ac2b895fd00fef40f4)

Scrolling seems to be strange, so instead, I let the texts break.